editor/doc_data.cpp

```diff
@@ -1169,10 +1188,18 @@ Error DocData::save_classes(const String &p_default_path, const Map<String, Stri
 
 		for (int i = 0; i < c.constants.size(); i++) {
 			const ConstantDoc &k = c.constants[i];
-			if (k.enumeration != String()) {
-				_write_string(f, 2, "<constant name=\"" + k.name + "\" value=\"" + k.value + "\" enum=\"" + k.enumeration + "\">");
+			if (k.is_value_valid) {
+				if (k.enumeration != String()) {
+					_write_string(f, 2, "<constant name=\"" + k.name + "\" value=\"" + k.value + "\" enum=\"" + k.enumeration + "\">");
+				} else {
+					_write_string(f, 2, "<constant name=\"" + k.name + "\" value=\"" + k.value + "\">");
+				}
 			} else {
-				_write_string(f, 2, "<constant name=\"" + k.name + "\" value=\"" + k.value + "\">");
+				if (k.enumeration != String()) {
+					_write_string(f, 2, "<constant name=\"" + k.name + "\" value=\"varies\" enum=\"" + k.enumeration + "\">");
+				} else {
+					_write_string(f, 2, "<constant name=\"" + k.name + "\" value=\"varies\">");
```

Comment:
I don't know if `varies` will be clear for the reader. How about `platform-dependent`?